### ISSUE TITLE
[Meta] Latest and Testing packwiz packs

### ISSUE THREAD

**Issue by RozeFound** [Fabulously-Optimized/fabulously-optimized#488]
1. Is it possible to make MultiMC (auto-update) for always latest version of minecraft?
Like, It will update modpack not only in terms of one minecraft version, but will update minecraft version itself.
2. If it is, so you can make Testing pack, for... Testers, with all of the mods that requires feedback.
These packs will also allow to avoid config loss on minecraft update.

**Comment by Julienraptor01** [comment]
So actually you can do update to minecraft versions manually with auto-update but at the risk of causing big problems
what you need to do is change the version in the pre-launch command
And MultiMC (auto-update) through minecraft versions wouldn't be done because by example actually we do support both 1.18.2 and 1.19.2
If we did update through minecraft versions it would mean we can't support multiple versions

**Comment by Madis0** [comment]
1. In theory, it wouldn't seem that hard to just copy the packwiz version to 1.19.2 folder and a "latest" folder. In practice, though...
   - What about alphas/betas? Having stable Minecraft one day and latest alpha with half the mods the next is not a pleasant experience. If I skipped alphas-betas altogether, then people would not get "the lastest", losing the point of the variant overall.
   - Let's assume there is a power user who usually likes even the alphas, but right now was just not ready yet. Then what, they'll try to downgrade on the same profile? I do not want to deal with downgraders and the amount of things that can break by that...
   - MultiMC (auto-update) variant is still clumsy in terms of disabling mods and its usage still doesn't contribute to the pack's development in any way. By contrast, even Modrinth users increase download count and therefore the popularity.
2. Ehh... Even with the assumption that I can just `packwiz mr install modname` and commit, I am not so sure it makes the testing easier for others while not making the work harder for me. 
For example, if I add 5 mods to testing and the user is saying that the performance is worse on testing pack compared to normal, how would I know which of those mods caused it?

To summarize, the ideas are nice but I'm not sure if _these solutions_ really solve the issues.
